x/leverage/types/params.go

```diff
@@ -67,3 +83,33 @@ func validateInterestEpoch(i interface{}) error {
 	}
 	return nil
 }
+
+func validateLiquidationThreshold(i interface{}) error {
+	v, ok := i.(sdk.Dec)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	if v.IsNegative() {
+		return fmt.Errorf("complete liquidation threshold cannot be negative: %d", v)
+	}
+	if v.GT(sdk.MustNewDecFromStr("0.1")) {
+		return fmt.Errorf("complete liquidation threshold cannot exceed 0.1: %d", v)
+	}
```

Comment:
```suggestion
```
Since this isn't hard limit (but strongly advisable) it may not belong in a Validation function.